getting-started/javascript.mdx

```diff
@@ -120,7 +120,7 @@ const ngrok = require('@ngrok/ngrok');
         traffic_policy: '{"on_http_request": [{"actions": [{"type": "oauth","config": {"provider": "google"}}]}]}'
     });
 
-    // Output ngrok url to console
+    // Output ngrok URL to console
```

Comment:
As a passing question, is there any way to make this ignore code snippets? In this case, it's a good change,  but I think it'd be weird if you got a vale error for a variable name